src/routes/docs/products/auth/security/+page.markdoc

```diff
@@ -40,3 +40,18 @@ These permissions are enforced for client SDKs and server SDKs when using JWT, b
 Creating and using mock phone numbers allows users to test SMS authentication without needing an actual phone number. This can be useful for testing edge cases where a user doesn't have a phone number but needs to sign in to your application using SMS. 
 
 To create a mock phone number, navigate to **Auth** > **Security** > Mock Phone Numbers. After defining a mock phone number, you need to define a specific OTP code that will be used for SMS sign-in instead of the SMS secret code sent to a real phone number.
+
+# Memberships privacy {% #memberships-privacy %}
+
+Hide specific membership details from other users by marking them as private. To set this up, go to **Auth** > **Security** > **Memberships privacy**
+
+These details can be made private:
+
+- `userName` - The member's name
+- `userEmail` - The member's email address
+- `mfa` - Whether the member has enabled multi-factor authentication
+
+{% info title="Scale plan feature" %}
+Memberships privacy is available on the Appwrite Cloud scale plan.
+{% /info %}
```

Comment:
```suggestion
```

We don't need this anymore since we're going to make this feature available to all plans.